### PR TITLE
Power management/hibernation

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
@apinter @Ethanol6 @Idesmi what else do we need to talk about on the topic of power-management for desktop users? `tlp`? Our wikis return no result and the Arch wikis are rather sparse about it.

### CODE REVIEW

**Comment by Ethanol6** [comment]
Is there a GUI tool that we can recommend alongside `tlp`?

**Comment by apinter** [comment]
Suspend/sleep to RAM could be mentioned, but TLP feels like a way too deep subject and they do offer a fairly thorough [upstream documentation](https://linrunner.de/tlp/). Not sure if we would need to document it again. (Arch has to document pretty much everything due to the nature of Arch as everything has to be pieced together by the user.)

**Comment by why-not-try-calmer** [comment]
> Suspend/sleep to RAM could be mentioned, but TLP feels like a way too deep subject and they do offer a fairly thorough [upstream documentation](https://linrunner.de/tlp/). Not sure if we would need to document it again. (Arch has to document pretty much everything due to the nature of Arch as everything has to be pieced together by the user.)

Right. You'd like to see more infos about suspend/sleep to RAM?

File: project/docs/power.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+You can finally lay back and admire your configuration. On my machine it looks like this:
+
+```
+# kernel parameter
+`resume=UUID=19bd024f-76bd-4162-ac37-4d0d51e02c25`
+```
+and 
+```
+# /etc/fstab
+UUID=e0bac415-cbde-4c9b-8178-7874ac9de70a swap defaults 0 0
```
Comment (teohhanhui) [inline project/docs/power.md:75]: Seems like it's missing `none` at the 2nd column, for the mount point...

Comment (Ethanol6) [reply project/docs/power.md:75]: Actually, on my system the second column says `swap` for the mount point. If this were a swap file and not a swap partition then you would be correct.

Comment (teohhanhui) [reply project/docs/power.md:75]: Be it a swap partition or a swapfile, it still wouldn't be mounted anywhere (hence `none`). `swap` is in the 3rd column, for the vfs type.

https://man7.org/linux/man-pages/man5/fstab.5.html

Comment (Ethanol6) [reply project/docs/power.md:75]: Ok, interesting. On my Tumbleweed systems this is not the case.
fstab_tumbleweed
This is the swap that the installer has made, not a manual creation like this document outlines.

Comment (why-not-try-calmer) [reply project/docs/power.md]: > Seems like it's missing `none` at the 2nd column, for the mount point...

You are correct, thanks, just fixed it.

**Comment by why-not-try-calmer** [comment]
Thanks for the grammar fix!

**Pull request merged**